recipes/qt/6.x.x/conanfile.py

```diff
@@ -775,7 +775,8 @@ def _create_plugin(pluginname, libname, type, requires):
             _create_module("WaylandClient", ["Gui", "wayland::wayland-client"])
             _create_module("WaylandCompositor", ["Gui", "wayland::wayland-server"])
 
-        self.cpp_info.components["qtCore"].cxxflags.append("-fPIC")
+        if self.settings.os != "Windows":
+            self.cpp_info.components["qtCore"].cxxflags.append("-fPIC")
```

Comment:
Why do you add -fPic only to qtCore? What about all other modules?

Reply:
all the other modules depend on qtCore, so they all get the flag for free

Reply:
Why is it ever added? It's an interface flag.

Reply:
this avoids falling into https://code.qt.io/cgit/qt/qtbase.git/tree/src/corelib/global/qglobal.h#n1394